src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/data/BasicUserDetail.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.capitalizeFully
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.repository.BasicUserPersonalDetail
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Basic User Information")
+data class BasicUserDetail(
```

Comment:
Struggling with the name here a bit. Not keen on the postfix `Detail` as it doesn't really add anything meaningful? Not sure what to suggest as an alternative though. If we're going with this I'd suggest renaming it to `UserBasicDetails`. I feel that reads a bit better.